app/src/main/java/org/oppia/app/walkthrough/WalkthroughFragmentChangeListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.walkthrough
+
+/** Listener for when an activity should change a fragment position. */
+interface WalkthroughFragmentChangeListener {
+  fun changeTo(pageNo: Int)
```

Comment:
Ditto for `pageNo` as previous comments.

Reply:
done